test/parallel/test-preload.js

```diff
@@ -8,9 +8,9 @@ var nodeBinary = process.argv[0];
 
 var preloadOption = function(preloads) {
   var option = '';
-  preloads.forEach(function(preload, index) {
-    // TODO: randomly pick -r or --require
-    option += '-r ' + preload + ' ';
+  preloads.forEach(function(preload) {
+    var optionString = ['-r', '--require'][Math.floor(Math.random()*2)];
```

Comment:
Wrap the `forEach` in `['-r', '--require'].forEach(function(optionString) {` and call it a day?
The test itself should be fast, so I don't see why this isn't worth doing?